yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@babel/runtime@^7.14.6":
   version "7.16.0"
-  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.16.0.tgz#e27b977f2e2088ba24748bf99b5e1dece64e4f0b"
+  resolved "https://registry.npmjs.org/@babel/runtime/-/runtime-7.16.0.tgz"
```

Comment:
I'm not totally sure, but I am wondering if you used `npm` to install the node packages instead of `yarn`? There could be something else at play here as well.

In any case, since `yarn.lock` is for JS dependencies and `sentry-raven` is a Ruby dependency, we shouldn't need to make any changes to this file as a part of this PR. I think a `git checkout origin/main -- yarn.lock` will undo these changes (which you'll need to then commit).

Reply:
I think I did use npm before I got yarn installed, which is probably an origin of a lot of my hiccups. I'll make sure to remove the `yarn-lock` changes. Thanks, Jim! Super helpful, much appreciated.